doc/whats_new/v0.21.rst

```diff
@@ -52,6 +52,15 @@ Support for Python 3.4 and below has been officially dropped.
   graph, which would add explicitly zeros on the diagonal even when already
   present. :issue:`12105` by `Tom Dupre la Tour`_.
 
+
+:mod:`sklearn.metrics`
+......................
+
+- |Feature| A new regression metric: :class:`metrics.max_error`: a
```

Comment:
this should be a sentence and you need to mention the scorer
see e.g., Added the metrics.balanced_accuracy_score metric and a corresponding 'balanced_accuracy' scorer for binary and multiclass classification. #8066 by @xyguo and Aman Dalmia, and #10587 by Joel Nothman.

Reply:
Updated. Also changed it from `:class:` to `:func:`